.github/workflows/ci.yml

```diff
@@ -76,10 +76,12 @@ jobs:
       - name: Smoke test ${{ matrix.os }} binary
         shell: bash
         run: |
-          target/debug/maker --data-dir=/tmp/maker --generate-seed testnet &
+          cp daemon/util/testnet_seeds/maker_seed /tmp/maker/maker_seed
+          target/debug/maker --data-dir=/tmp/maker testnet &
           sleep 10s # Wait for maker to start
 
-          target/debug/taker --data-dir=/tmp/taker --generate-seed testnet &
+          cp daemon/util/testnet_seeds/taker_seed /tmp/taker/taker_seed:
+          target/debug/taker --data-dir=/tmp/taker --maker-id 10d4ba2ac3f7a22da4009d813ff1bc3f404dfe2cc93a32bedf1512aa9951c95e testnet &
```

Comment:
will we need to manually upgrade this key, increasing the maintenance burden?

Reply:
if we change the seeds in the repo, yes